docs/user_stories/loadflow_tutorial.md

```diff
@@ -0,0 +1,65 @@
+---
+title: User story number 1
+layout: default
+---
+
+# Easy network to start with PowSyBl
+
+In this user story, the network is quite simple and made of two lines in parallel, with on left side a generator and on right side a load. The load consumes 600 MW and the generator produces 606.5 MW. A load flow computes the flows on the two lines. The flow on the upper line is of 302.4 MW at its entrance and of 300.4 MW at its exit. The flow in the lower line is the same. The power losses are of 2 MW on each line.   
+
+![Eurostag network initial](./images/Network_Eurostag_Initial.svg){: width="50%" .center-image}
+
+
+In this tutorial, we apply a contingency: the upper line is disconnected. A new load flow computes the flow on the lower line: it is now of 610.6 MW at its entrance and of 601 MW at its exit. The rest of the difference between load and generation represents the losses during the voltage transformation process.
+
+![Workflow](./images/Network_Eurostag_Final.svg){: width="50%" .center-image}
+
+# Workflow
+
+The tutorial can be translated in a short and easy workflow. All the input data are stored in a XIIDM file. This file is imported with the IIDM gateway. Then, a load flow simulator is launched to get flows on all nodes. In this tutorial, the simulator is Hades2 but could be an other load flow simulator, as the API interface contract is respected. A contingency is created and finally, the flows are computed again in order to get the final state.  
+
+![Eurostag network final](./images/Workflow.svg){: width="75%" .center-image}
+
+# Identification of the power system blocks
+
+<img src="./images/File.svg" alt="" style="vertical-align: bottom"/>
+The network is modeled in [IIDM](../iidm/model/index.md), which is the internal model of PowSyBl. This model can be serialized in a XML format for experimental purposes.
+```java
+File file = new File(<path_to_file_"eurostag-tutorial1-lf.xml">);
```

Comment:
File file = new File("/path/to/file/eurostag-tutorial1-lf.xml");

We should add a link to help user to download this file. It could be found in the resources of powsybl-core (XML converter I guess)

Reply:
In fact, in the tutorial the path is  
```java
File file = new File(LoadflowTutorial.class.getResource("/eurostag-tutorial1-lf.xml").getPath());
```
Should I put this line here ?

Reply:
No sure it will works properly once packaged as a jar. I think we should keep it as a file path like the others